src/core/sys/posix/ucontext.d

```diff
@@ -765,6 +765,38 @@ else version( FreeBSD )
             int[6]          mc_spare2;
         }
     }
+    else version( AArch64 )
+    {
+        alias long __register_t;
```

Comment:
Can we please use the new alias syntax? (will correct this quickly myself)

Reply:
> ERROR: Permission to myfreeweb/druntime.git denied to wilzbach.

Oh well, it looks like you need to do it yourself or enable maintainer access to your PRs.

Reply:
Oh the one with the `=` sign is new? I'm not familiar with D, this is the first D code I'm editing :) I assumed this is the new one since it's more common in that file.

Reply:
yes, `alias __register_t = long` is the "new alias syntax".
Unfortunately the old one was never deprecated and as there's still some code that uses it (druntime being a prime example).
We try to avoid using it for new code at least ;-)
Thanks for the fix-up!